src/index.html

```diff
@@ -2,15 +2,48 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./fonts/Avenir-Book.ttf">
+    <link rel="stylesheet" href="./fonts/Avenir-Heavy.ttf">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    data-qa="big"
+    class="big__form"
+    action="/"
+    method="post"
+    >
+      <img
+        class="big__img"
+        src="./images/Search.svg"
+        alt="Search"
+      >
+      <input
+        class="big__input"
+        data-qa="keypress"
+        placeholder="Try &quot;Los Angeles&quot;"
+      >
+    </form>
+
+    <form
+    data-qa="small"
+    class="small__form"
+    action="/"
+    method="post"
+    >
```

Comment:
```suggestion
    <form
      data-qa="small"
      class="small__form"
      action="/"
      method="post"
    >
```

Dont forget about correct indentation

Reply:
i'm forget this do thanks for help))